pkg/source/vmware/client.go

```diff
@@ -402,31 +401,18 @@ func (c *Client) GenerateVirtualMachine(vm *migration.VirtualMachineImport) (*ku
 		})
 	}
 
-	// setup bios/efi, secureboot and tpm settings
-
-	if o.Config.Firmware == "efi" {
-		firmware := &kubevirt.Firmware{
-			Bootloader: &kubevirt.Bootloader{
-				EFI: &kubevirt.EFI{
-					SecureBoot: &boolFalse,
-				},
-			},
-		}
-		if *o.Config.BootOptions.EfiSecureBootEnabled {
-			firmware.Bootloader.EFI.SecureBoot = &boolTrue
-			vmSpec.Template.Spec.Domain.Features.SMM = &kubevirt.FeatureState{
-				Enabled: &boolTrue,
-			}
-		}
-		vmSpec.Template.Spec.Domain.Firmware = firmware
-		if *o.Summary.Config.TpmPresent {
-
-			vmSpec.Template.Spec.Domain.Devices.TPM = &kubevirt.TPMDevice{}
-		}
+	// Setup BIOS/EFI, SecureBoot and TPM settings.
+	uefi := o.Config.Firmware == "efi"
```

Comment:
should `efi` be a const, and `o.Config.Firmware` is converted to lower-case and then comparing?

Reply:
Done